exercises/anagram/test/Tests.hs

```diff
@@ -14,9 +15,14 @@ specs = describe "anagram" $
           describe "anagramsFor" $ for_ cases test
   where
 
-    test Case{..} = it description $ expression `shouldBe` expected
+    test Case{..} = it description $ expression `shouldMatchList` expected
       where
-        expression = anagramsFor subject candidates
+        expression = map toList
+                   . toList
+                   . anagramsFor (fromList subject)
+                   . fromList
+                   . map fromList
```

Comment:
[IsList](http://hackage.haskell.org/package/base-4.9.0.0/docs/GHC-Exts.html#t:IsList) is fantastic. Way more flexible than I imagined. :smile: